arb_os/output.mini

```diff
@@ -227,6 +238,8 @@ impure func emitBlockSummaryLog(blockNum: uint, timestamp: uint, prevBlockNum: u
         globalBlockReceiptData.statsAllTime,
         prevBlockNum
     ),) { log };
+    asm(merkleTreeBuilder_rootHash(merkleOfSends),) { send };
```

Comment:
As discussed offline, we may want to reduce the frequency of sending Merkle roots depending on the cost of processing a single Merkle root